cmd/bundle.go

```diff
@@ -118,17 +135,72 @@ func runBundleNew(cmd *cobra.Command, args []string) error {
 		// Promptui templates are a nightmare. Need to support multi repos when moving this to bubbletea
 		TemplateRepo: "/massdriver-cloud/application-templates",
 		// TODO: unify bundle build and app build outputDir logic and support
-		OutputDir: ".",
+		OutputDir: outputDir,
 	}
 
 	err = bundle.RunPromptNew(templateData)
-
 	if err != nil {
 		return err
 	}
 
 	err = commands.GenerateNewBundle(cache, templateData)
```

Comment:
If we move the if else in to runBundleNew, after the if else you can invoke this because we do the exact same thing once we have handled the input data.

something like 
```
if nonInteractiveMode(cmd) {
  doNonInteractiveStuff()
} else {
  doInteractiveStuff()
}

makeTheTemplate(templateData)
```